guides/common/assembly_using-foreman-webhooks.adoc

```diff
@@ -0,0 +1,37 @@
+ifdef::context[:parent-context: {context}]
+
+[id="using-foreman-webhooks_{context}"]
+= Using Foreman Webhooks
+
+A webhook is a way for a web page or web application to provide other applications with information in real time. 
+Webhooks are only triggered after an event occurs. 
+For example, when a user places an online order, a webhook makes an HTTP request to a pre-configured URL. 
```

Comment:
Can you remove this example as you have explained it in a Foreman context just below and this is not needed. It would be great to incorporate the  `a webhook makes an HTTP request to a pre-configured URL.` into the example below.